Detections/AlsidForAD/PasswordIssues.yaml

```diff
@@ -0,0 +1,28 @@
+id: 472b7cf4-bf1a-4061-b9ab-9fe4894e3c17
+name: Alsid Password issues
+description: |
+    'Searches for triggered Indicators of Exposures related to password issues'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AlsidForAD
+    dataTypes:
+      - AlsidForADLog_CL
+queryFrequency: 2h
+queryPeriod: 2h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+  let SeverityTable=datatable(Severity:string,Level:int) [
+  "low", 1,
+  "medium", 2,
+  "high", 3,
+  "critical", 4
+  ];
+  afad_parser
+  | where MessageType == 0 and (Codename == "C-CLEARTEXT-PASSWORD" or Codename == "C-PASSWORD-DONT-EXPIRE" or Codename == "C-USER-REVER-PWDS" or Codename == "C-PASSWORD-POLICY" or Codename == "C-USER-PASSWORD" or Codename == "C-KRBTGT-PASSWORD" or Codename == "C-AAD-SSO-PASSWORD" or Codename == "C-REVER-PWD-GPO")
```

Comment:
Change to List, same as I had shown in "Alsid Active Directory attacks pathways" detection.